packages/ui/src/components/UnreadDot.tsx

```diff
@@ -0,0 +1,17 @@
+import { ComponentProps } from 'react';
+
+import { Circle } from '../core';
+
+export function UnreadDot(
+  props: ComponentProps<typeof Circle> & { color?: 'primary' | 'neutral' }
+) {
+  return (
+    <Circle
+      size="$m"
+      backgroundColor={
+        props.color === 'neutral' ? '$gray300' : '$positiveActionText'
```

Comment:
Should be theme color -- usually even grayscale color change from dark to light me.